src/BadgeButton.webmodeler.ts

```diff
@@ -34,23 +32,24 @@ export class preview extends Component<BadgeButtonContainerProps, {}> {
         return {
             bootstrapStyle: props.bootstrapStyle,
             className: props.class,
-            label: props.label,
+            label: props.label || "Button",
             style: BadgeButtonContainer.parseStyle(props.style),
-            value: valueAttributeArray ? "[" + valueAttributeArray + "]" : props.badgeButtonValue || " "
+            value: valueAttributeArray ? "[" + valueAttributeArray + "]" : props.badgeButtonValue || "Badge"
```

Comment:
This the incorrect way of providing the default values.
This should not done here but in the widget.xml as a translatable string